Cargo.toml

```diff
@@ -6,16 +6,17 @@ authors = ["Apollo Devs"]
 edition = "2021"
 repository = "https://github.com/apollodao/cw-token"
 
+[workspace]
```

Comment:
Why is this needed?

Reply:
This is need it for crate optimizoor to compile and generate wasm files, since the testing framework clone from github the branch and uses as input to the compiler, but the integration test works as a workspace and rust complains that the cloned repo doesnt belongs to him, except if you **explicit** put an empty workspace.